lms/static/js/learner_dashboard/views/program_header_view.js

```diff
@@ -0,0 +1,56 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'text!../../../templates/learner_dashboard/program_header_view.underscore',
+            'picturefill'
+           ],
+         function(Backbone, $, pageTpl, picturefill) {
+            return Backbone.View.extend({
+                breakpoints: {
+                    min: {
+                        'medium': '768px',
+                        'large': '1180px',
+                        'x-large': '1440px'
```

Comment:
@rlucioni The break points for program listing page are different from break points on this page. The banner image on this page spread across the whole page. The banner image on listing page are limited to the cards.